packages/cli/package.json

```diff
@@ -29,7 +29,7 @@
     "lint": "yarn run --top-level eslint -c .eslintrc.js ",
     "prepublish": "",
     "prepack": "yarn run build && oclif manifest && oclif readme",
-    "test": "TZ=UTC NODE_OPTIONS=--experimental-vm-modules yarn jest --runInBand --forceExit"
+    "test": "TZ=UTC NODE_OPTIONS=--experimental-vm-modules yarn jest --runInBand --forceExit reserve/transfergold.test.ts releasecelo/withdraw.test.ts"
```

Comment:
remove before merge i suspect

Reply:
yeah forgot to mark it as a draft, it's only temporarily here to make it easier for @arthurgousset to debug